### PR TITLE
start button and toast message added

### PR DESCRIPTION
**Pull request by selva-tuva**

### CODE REVIEW

**Review by jpatel3 — APPROVED** [review]
looks fine.

File: inapp_survey/templates/inapp_survey/index.html [hunk 1]
```diff
@@ -130,6 +293,16 @@
       isAuthenticated: '{{user.is_authenticated|yesno}}'==='yes'
   };
 </script>
+<script>
+  // To check background MATCH
+  setInterval(
+    function () {
+      var randomColor = Math.floor(Math.random() * 16777215).toString(16);
+      document.body.style.backgroundColor = "#" + randomColor;
```
Comment (naveenkumarganesan) [inline inapp_survey/templates/inapp_survey/index.html]: @selva-tuva Wy are we changing the background color of body?

**Comment by jpatel3** [comment]
is it ever been merged?

**Comment by naveenkumarganesan** [comment]
yes @jpatel3

**Pull request closed without merging**